src/ast.ts

```diff
@@ -1,6 +1,7 @@
 // Dependencies:
 import { createSourceFile, ScriptTarget, SourceFile } from 'typescript';
 
-export function createAST (source: string, fileName?: string): SourceFile {
-    return createSourceFile(fileName || '', source, ScriptTarget.Latest, true);
+// ScriptKind values : https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L4626
+export function createAST (source: string, fileName?: string, scriptKind?: number): SourceFile {
+    return createSourceFile(fileName || '', source, ScriptTarget.Latest, true, scriptKind);
```

Comment:
Instead of `number` could this use `ScriptKind`? Then you can remove the comment too.